.travis.yml

```diff
@@ -36,8 +36,7 @@ matrix:
   include:
 
     - compiler: gcc
-      env: GCC_VER=5 TARGET=debug
-      # - APP_ARGS="--unittest-jobs=2"
+      env: GCC_VER=5 TARGET=debug APP_ARGS="--unittest-ipv4"
```

Comment:
Ubuntu build-and-test.sh puts a space between the opening quote and the `--unittest`.  I'm wondering if that might be appropriate here, if only to help with future maintenance?  Just a thought.